File: app/src/main/java/aromko/de/wishlist/model/UserSetting.java
package aromko.de.wishlist.model;

public class UserSetting {

    private String favoriteListId;

    public UserSetting() {
    }

    public UserSetting(String favoriteListId) {
        this.favoriteListId = favoriteListId;
    }

    public String getFavoriteListId() {
        return favoriteListId;
    }

    public void setFavoriteListId(String favoriteListId) {
        this.favoriteListId = favoriteListId;
    }

    @Override
    public String toString() {
        return "UserSettings{" +
                "favoriteListId='" + favoriteListId + '\'' +
                '}';
    }
}

File: app/src/main/java/aromko/de/wishlist/model/UserSetting.kt
package aromko.de.wishlist.model

class UserSetting {
    var favoriteListId: String? = null

    constructor()
    constructor(favoriteListId: String?) {
        this.favoriteListId = favoriteListId
    }

    override fun toString(): String {
        return "UserSettings{" +
                "favoriteListId='" + favoriteListId + '\'' +
                '}'
    }
}